internal/cli/users.go

```diff
@@ -22,8 +66,341 @@ func usersCmd(cli *cli) *cobra.Command {
+auth0 users create --name "John Doe" 
+auth0 users create -n "John Doe" --email john@example.com`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			// Select from the available connection types
+			// Users API currently support  database connections
+			if err := userConnection.Select(cmd, &inputs.Connection, cli.connectionPickerOptions(), nil); err != nil {
+				return err
+			}
+
+			// Prompt for user's name
+			if err := userName.Ask(cmd, &inputs.Name, nil); err != nil {
+				return err
+			}
+
+			// Prompt for user email
+			if err := userEmail.Ask(cmd, &inputs.Email, nil); err != nil {
+				return err
+			}
+
+			// Prompt for user password
+			if err := userPassword.Ask(cmd, &inputs.Password, nil); err != nil {
+				return err
+			}
+
+			// The getConnReqUsername returns the value for the requires_username field for the selected connection
+			// The result will be used to determine whether to prompt for username
+			conn := cli.getConnReqUsername(auth0.StringValue(&inputs.Connection))
+			requireUsername := auth0.BoolValue(conn)
+
+			var a *management.User
+
+			// Prompt for username if the requireUsername is set to true
+			// Load values including the username's field into a fresh users instance
+			// Else block loads values without username for connections with requireUsername set to false
+			if requireUsername {
+				if err := userUsername.Ask(cmd, &inputs.Username, nil); err != nil {
+					return err
+				}
+				a = &management.User{
+					Connection:    &inputs.Connection,
+					Email:         &inputs.Email,
+					Name:          &inputs.Name,
+					Username:      &inputs.Username,
+					Password:      &inputs.Password,
+				}
+			} else {
+				a = &management.User{
+					Connection:    &inputs.Connection,
+					Email:         &inputs.Email,
+					Name:          &inputs.Name,
+					Password:      &inputs.Password,
+				}
+			}
+
+
+			// Create app
+			if err := ansi.Waiting(func() error {
+				return cli.api.User.Create(a)
+			}); err != nil {
+				return fmt.Errorf("Unable to create user: %w", err)
+			}
+
+			// Render Result
+			cli.renderer.UserCreate(a)
+
+			return nil
+		},
+	}
+	userName.RegisterString(cmd, &inputs.Name, "")
+	userConnection.RegisterString(cmd, &inputs.Connection, "")
+	userPassword.RegisterString(cmd, &inputs.Password, "")
+	userEmail.RegisterString(cmd, &inputs.Email, "")
+	userUsername.RegisterString(cmd, &inputs.Username, "")
+
+	return cmd
+}
+
+func showUserCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "show",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Show an existing user.",
+		Long:  "Show an existing user.",
+		Example: `auth0 users show 
+auth0 users show <id>`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
auth0 users show <id>`,
		RunE: func(cmd *cobra.Command, args []string) error {
```
`prepareInteractivity` is now on the root command.